.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+_Describe the problem or feature in addition to a link to the issues._
+
+Before submitting this PR, please make sure:
+
+- [ ] You have added a few sentences describing the PR here.
+- [ ] You have added yourself or the appropriate individual as the assignee.
+- [ ] You have added at least one relevant code reviewer to the PR.
+- [ ] Your code builds clean without any errors or warnings.
```

Comment:
Might want to rephrase this, since WDL doesn't actually build. I'd specify that it passes `sprocket lint`?